CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ PostgreSQL 16 support will be added with a following TimescaleDB release.
 * #6053 Fix approximate_row_count for CAggs
 * #6081 Improve compressed DML datatype handling
 * #6084 Propagate parameter changes to decompress child nodes
+* #6102 Schedule compression policy more often
```

Comment:
I guess `#6106 incorrect pointer usage in OSM hook callback` is also part of 2.12.0

Reply:
It's an internal change, thats why its not part of changelog